.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - '8'
+
+notifications:
+  email: false
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+script:
+  - yarn lint
+  - yarn test
```

Comment:
We're using jest runners, so you can just do `yarn run jest --projects jest.*.config.js`.

PS: the current `yarn lint` command is actually wrong, it should be `jest --projects jest.eslint.config.js jest.stylelint.config.js`

Reply:
looks like travis took care of this?

```
$ yarn lint
yarn run v1.3.2
$ jest --projects jest.*.config.js
```

Reply:
> the current yarn lint command is actually wrong
👆